common/common.c

```diff
@@ -1288,6 +1288,35 @@ void load_font_glyph(const char *program, lv_obj_t *element) {
     }
 }
 
+void load_font_section(const char *program, const char *section, lv_obj_t *element) {
```

Comment:
I went with a 'section' parameter as it allows this method to be easily re-used in the future if we want to have more font configurations like for headers, and footers or whatever